src/status_im/contexts/communities/actions/community_rules/view.cljs

```diff
@@ -1,14 +1,19 @@
 (ns status-im.contexts.communities.actions.community-rules.view
   (:require
+    [quo.components.buttons.slide-button.utils :as slide-button-utils]
     [quo.core :as quo]
     [react-native.core :as rn]
+    [react-native.platform :as platform]
     [status-im.contexts.communities.actions.community-rules.style :as style]
     [utils.re-frame :as rf]))
 
 (defn view
   [id]
-  (let [rules (rf/sub [:communities/rules id])]
-    [rn/view {:style style/community-rule}
+  (let [rules                 (rf/sub [:communities/rules id])
+        android-bottom-margin (if platform/android? 44 0)
```

Comment:
Because the dimensions of the slider change according to the screen dimensions. Which causes different behave since android does not have safe area

Reply:
This should be using safe area then.

Reply:
> Why additional margin is required for Android?

The bottom margin on Android is a bit too big indeed. I didn't report this now to avoid wasting time on this minor visual inconsistency in an edge case, since this issue itself is a release blocker and the margin could be fixed later. 

What do you think guys about fixing this as a follow-up? @ajayesivan @J-Son89 @FFFra 

But if this is a quick one, then it would be nice to fix it today for sure

Reply:
@J-Son89 Yah, I tried safe area, but it has no effect in Android. Ideally we need a solution for all cases. Please let me know if any ideas

Reply:
Hmm is there a safe area elsewhere pushing this down?

Seems like you might need to remove something rather than adding it 🤔

Reply:
Actually you are right. I made another change and the other problem was a `position: absolute` ruining our lifes hahaha